rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2022-02-14
```

Comment:
This is a bad idea. A library should not bind to a specific version.

Reply:
ok, remove it